sdk/devcenter/developer-devcenter-rest/test/public/devBoxesTest.spec.ts

```diff
@@ -15,14 +15,12 @@ import {
 describe("DevCenter Dev Boxes Operations Test", () => {
   let recorder: Recorder;
   let client: AzureDevCenterClient;
-  let tenantId: string;
-  let devCenter: string;
+  let endpoint: string;
 
   beforeEach(async function (this: Context) {
     recorder = await createRecorder(this);
-    tenantId = env["DEVCENTER_TENANT_ID"] || "<tenant id>";
-    devCenter = env["DEFAULT_DEVCENTER_NAME"] || "sdk-default-devcenter";
-    client = createRecordedClient(recorder, tenantId, devCenter, {
+    endpoint = env["DEVCENTER_ENDPOINT"] || "<endpoint>";
```

Comment:
nitpick: better to use the same environment variable name as the one in the samples?